### ISSUE TITLE
Support $BuiltModuleSubdirectory configured in build.yaml

### ISSUE THREAD

**Issue by NicolasBn** [dsccommunity/DscResource.DocGenerator#82]
#### Details of the scenario you tried and the problem that is occurring
When `BuiltModuleSubDirectory` set in configuration file build.yaml, the value is ignored by the tasks.

(cf. https://github.com/gaelcolas/Sampler/issues/299)
#### Steps to reproduce the problem

Set `BuiltModuleSubDirectory` on build.yaml.

#### Expected behavior
`$BuiltModuleSubDirectory` takes the value in build.yaml.
#### Current behavior
`$BuiltModuleSubDirectory` doens't take the value in build.yaml.
#### Suggested solution to the issue

Modify all the tasks as same as Set-SamplerTaskVariable.ps1 in sampler : 
https://github.com/gaelcolas/Sampler/blob/8d74f0c84d3d012ebbd4b61b5d2c243b93d8f05c/Sampler/scripts/Set-SamplerTaskVariable.ps1#L89

Or dotsource Set-SamplerTaskVariable.ps1 as like DSCResource.Test : 
https://github.com/dsccommunity/DscResource.Test/blob/d2ced8246c29f6b56bf7926cedb1d81d1ea9c752/source/tasks/Fail_Build_If_HQRM_Tests_Failed.build.ps1#L61

**Comment by johlju** [comment]
I think when this issue is resolved it will also close issue #19. 🤔

**Comment by gaelcolas** [comment]
Example of the error here: https://synedgy.visualstudio.com/nxTools/_build/results?buildId=1386&view=logs&j=5d0a9c4e-8741-5118-af45-406a30fe661c&t=bda0b158-a237-516f-c1d0-31339587c5b5&l=157


```
			PUBLISH GITHUB WIKI CONTENT
This task publishes documentation to a GitHub Wiki repository.
-------------------------------------------------------------------------------
  /publish/Publish_GitHub_Wiki_Content
  /home/vsts/work/1/s/output/RequiredModules/DscResource.DocGenerator/0.10.1/tasks/Publish_GitHub_Wiki_Content.build.ps1:102

	OutputDirectory       = '/home/vsts/work/1/s/output'
ERROR: Cannot find path '/home/vsts/work/1/s/output/nxtools' because it does not exist.
At /home/vsts/work/1/s/output/RequiredModules/DscResource.DocGenerator/0.10.1/tasks/Publish_GitHub_Wiki_Content.build.ps1:152 char:41
+ … ModuleManifest = [string](Get-Item -Path $builtModuleManifest).FullNa …
+                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
